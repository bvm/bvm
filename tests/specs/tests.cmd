@echo off

CALL bvm install --use https://bvm.land/deno/1.15.2.json@affc933ab2513eb554f3188dee15bee19b0c97d171138384d31b611dafe679a2
CALL deno eval "console.log(\"hello\")"
CALL deno eval "console.log(2 != 3)"
REM cmd supports no quotes here, but powershell and cmd both require them
CALL deno eval -p JSON.stringify({})
CALL deno eval -p "JSON.stringify({})"
CALL deno eval "console.log(Deno.args[0])" lib=""
CALL deno eval "console.log(Deno.args[0])" -- lib=test,other
