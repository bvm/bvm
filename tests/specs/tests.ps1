& bvm install --use https://bvm.land/deno/1.15.2.json@affc933ab2513eb554f3188dee15bee19b0c97d171138384d31b611dafe679a2
# for some reason, double quotes within double quotes escaped with backticks wasn't even working with deno.exe
& deno eval 'console.log(`"hello`")'
& deno eval "console.log(2 != 3)"
& deno eval -p "JSON.stringify({})"
& deno eval "console.log(Deno.args[0])" lib=""
& deno eval "console.log(Deno.args[0])" -- lib=test,other
