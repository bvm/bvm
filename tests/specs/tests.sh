#!/bin/sh

bvm install --use https://bvm.land/deno/1.15.2.json@affc933ab2513eb554f3188dee15bee19b0c97d171138384d31b611dafe679a2
deno eval "console.log(\"hello\")"
deno eval "console.log(2 != 3)"
deno eval -p "JSON.stringify({})"
deno eval "console.log(Deno.args[0])" lib=""
deno eval "console.log(Deno.args[0])" -- lib=test,other