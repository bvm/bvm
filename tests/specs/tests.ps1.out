hello
true
{}
lib=
lib=test,other